test/integration/targets/iam_oidc_federation/tasks/main.yml

```diff
@@ -0,0 +1,169 @@
+- module_defaults:
+    group/aws:
+      region: "{{ aws_region }}"
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+  block:
+    - name: Set facts for tests
+      set_fact:
```

Comment:
Personally I'd set these in defaults rather than in the block...

Reply:
Will do